lib/ontologies_linked_data/serializers/json.rb

```diff
@@ -40,13 +44,33 @@ def self.serialize(obj, options = {})
             context = {"@context" => context_hash}
             hash.merge!(context)
           end
-          hash['@context']['@language'] = options[:lang] if hash['@context']
+          hash['@context']['@language'] = result_lang if hash['@context']
         end
         MultiJson.dump(hash)
       end
 
       private
 
+      def self.get_languages(submission, user_languages)
+        
+        if submission
+          
+          submission.bring :naturalLanguage
+          langauges = get_submission_languages(submission.naturalLanguage)
+                  
+          # intersection of the two arrays , if the requested language is not :all
+          result_lang = user_languages == :all ? langauges : user_languages & langauges
```

Comment:
Hi @haddadzineddine it seems that issue is here 
I got this error running it locally, I let you fix it.
<img width="1563" alt="image" src="https://github.com/ontoportal-lirmm/ontologies_linked_data/assets/29259906/930d9d95-b4d6-4487-b312-b15ab35fb66d">

Reply:
How can i reproduce this error ?